packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -60,23 +64,38 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
         setShouldShowPopup(false);
     };
 
+    const onHandleReturnFnDesktop = () => {
+        my_profile_store.setIsCancelAddPaymentMethodModalOpen(true);
+    };
+
     const setSubmitForm = submitFormFn => (submitForm.current = submitFormFn);
 
     React.useEffect(() => {
         if (!should_show_popup) {
             setErrorMessage(null);
         }
 
-        my_profile_store.setShouldShowAddPaymentMethodForm(false);
         my_profile_store.setSelectedPaymentMethod('');
         my_profile_store.setSelectedPaymentMethodDisplayName('');
 
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [should_show_popup]);
 
+    const paymentMethodTitle = () => {
+        if (!isMobile()) {
+            return (
+                <>
+                    <Icon icon='IcArrowLeftBold' onClick={onHandleReturnFnDesktop} />
+                    &nbsp;{localize('Add payment method')}
+                </>
+            );
+        }
+        return localize('Add payment method');
+    };
+
     const Form = general_store.nickname ? BuySellForm : NicknameForm;
     const modal_title = my_profile_store.should_show_add_payment_method_form
-        ? localize('Add payment method')
+        ? paymentMethodTitle()
         : table_type === buy_sell.BUY
         ? localize('Buy {{ currency }}', { currency: selected_ad.account_currency })
         : localize('Sell {{ currency }}', { currency: selected_ad.account_currency });
```

Comment:
Since you're adding a function to get the title can we remove this nested ternary and add it to the function as well? It'll make our eslint happier 😄

Reply:
Got it!